### ISSUE TITLE
Campaign: Edit flyer location

### ISSUE THREAD

**Issue by nscharioth** [verdigado/gruene_app#262]
When clicking on one of the existing poster icons in #218 #261, a popup appears with further information.

Features:
- [ ] Number of flyers given out at location
- [ ] "Infos & Bearbeiten" leads to new popup:
- [ ] Edit address
- [ ] Edit number of flyers
- [ ] Delete location
- [ ] Cancel 
- [ ] Save

<img width="317" alt="Bildschirmfoto 2024-08-25 um 13 04 36" src="https://github.com/user-attachments/assets/d5aaee81-7097-46b6-8979-f5683e3a1706">
<img width="318" alt="Bildschirmfoto 2024-08-25 um 14 21 07" src="https://github.com/user-attachments/assets/2facc0c9-a1ea-41fd-89ea-b1b722831ee7">
<img width="335" alt="Bildschirmfoto 2024-08-25 um 14 21 51" src="https://github.com/user-attachments/assets/495336ef-b0e6-4f4a-922f-c8ff444010c0">

Prototype:
https://www.figma.com/proto/7wE0Iveas3GGb8u4oL0714/App-Design-Mitglieder-App-(1)-(Copy)?node-id=2670-9749&t=b8swD5IOAFfZ0OSo-1&starting-point-node-id=2425%3A17429

Pollion reference: https://github.com/pollion/sunflower-app/issues/31

**Comment by nscharioth** [comment]
Please review issue description and move to "Ready for Development" column. @leagerndt @JuliaFreitagBGSt

**Comment by JuliaFreitagBGSt** [comment]
@nscharioth we are not able to review the both screenshots. Please clarify.

Issue is reviewed.

**Comment by nscharioth** [comment]
> [@nscharioth](https://github.com/nscharioth) we are not able to review the both screenshots. We assume they are screenshots of the prototype regarding the described issue. Please confirm.
> 
> Issue is reviewed.

Yes, they are screenshots from the prototype for this function. 
Click on an existing blue icon and press on "Infos & Bearbeiten".
